eng/pipelines/templates/steps/select-feed.yml

```diff
@@ -0,0 +1,16 @@
+parameters:
+  # Publish to https://dev.azure.com/azure-sdk/public/_packaging?_a=feed&feed=azure-sdk-for-net
+  DevOpsFeedId: '29ec6040-b234-4e31-b139-33dc4287b756/fa8c16a3-dbe0-4de2-a297-03065ec1ba3f'
+
+steps:
+- pwsh: |
+    # For safety default to publishing to the private feed.
+    # Publish to https://dev.azure.com/azure-sdk/internal/_packaging?_a=feed&feed=azure-sdk-for-net-pr
```

Comment:
Can we just use `Build.Repository.Name` in the URL?

Reply:
Oh, it's based on Id. got it.

Reply:
This is how AzDO does internal feeds - that's just the URL it resolves to. It has to be the tenant and feed ID.